.github/actions/setup-dependencies.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+source /etc/lsb-release
+
+# TODO: Add setup for Ubuntu 20.04
```

Comment:
Shouldn't the packages be the same?  I have never known the packages to differ.

Reply:
Also shouldn't we check for non-ubuntu systems like redhat?

Reply:
> Shouldn't the packages be the same? I have never known the packages to differ.

My thought was to give the ability to install extra packages (i.e. clang analysis tools) for 20.04 that we won't be running on 18.04 for the quality check. On the other hand, I could just add that as a step in the quality job so maybe that's overkill.

> Also shouldn't we check for non-ubuntu systems like redhat?

Nope, I only intend for this to be used to setup github actions and I don't think we're planning on testing on redhat lineage systems anytime soon.

Reply:
@Joshua-Anderson let us add that as an extra step in the quality job.

Reply:
Could you go ahead and just make this install packages without release checking.  That way we don't need to update this unless the ubuntu package structure changes on us.